packages/fast-components/src/slider/slider-utilities.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Converts a pixel coordinate on the track to a percent of the track's range
+ */
+export function convertPixelToPercent(
+    pixelPos: number,
+    minPosition: number,
+    maxPosition: number,
+    direction?: string
+): number {
+    let pct: number = 0;
```

Comment:
there is a `limit` function in `@microsoft/fast-web-utilities` you can leverage: 
```const pct = limit(0, 1, (pixelPos - minPosition) / maxPosition);```

Reply:
done, thanks!